docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3965,7 +3965,7 @@ List nodes
 ### Inspect a node
 
 
-`GET /nodes/<id>`
+`GET /nodes/(id or name)`
```

Comment:
We should not include this in the 1.24 documentation, given that docker 1.12 will not support this.